converter/prototype.py

```diff
@@ -148,13 +148,16 @@ def get_destination_settings_if_any(
         case "INTERNAL_NODE", None:
             destination = None
         case "INTERNAL_NODE", transition_node_id:
-            destination = utils.gen_object_id(transition_node_id)
-            transition_node = context.fig_node(transition_node_id)
-
-            if "overlayBackgroundInteraction" in transition_node:
-                overlay_settings = FlowOverlaySettings.Positioned(
-                    transition_node.get("overlayPositionType", "CENTER")
-                )
+            if transition_node_id[0] == 4294967295:
```

Comment:
Maybe we can have it as a constant or add a comment to avoid de magic number? Also, do you think a small test for this case could be worth it?

Btw, I can't say I understand how this relates to group's sizing constraints 🙈

Reply:
Ah, this is another thing. This PR is actually three different fixes :). I had a comment explaining the changes but GH ate it.

It's the:
- Group size constraint.
- This one (any reference with UINT_MAX must be ignore, this is just another case we missed)
- The change here when I was deleting keys in style overrides for no good reason: https://github.com/sketch-hq/fig2sketch/pull/52/files#diff-b518a19cbe208477554b46c18ac70c17e18e9a2e90d386387f36a6cb33bd2945L132

Reply:
BTW, I just extracted this to a function and replaced it in other places. Take a look! :pray:

Reply:
Oh, love it!!